pkg/storage/storagebase/proposer_kv.proto

```diff
@@ -92,7 +92,8 @@ message ReplicatedEvalResult {
   bool is_consistency_related = 9;
   // The stats delta corresponding to the data in this WriteBatch. On
   // a split, contains only the contributions to the left-hand side.
-  storage.engine.enginepb.MVCCNetworkStats delta = 10 [(gogoproto.nullable) = false];
+  storage.engine.enginepb.MVCCStats old_delta = 10;
```

Comment:
`deprecated_delta`? and a pointer at the version which deprecates it.

Reply:
Yeah, that's a better name. Done.